packages/plugin-lighthouse/src/lib/utils.unit.test.ts

```diff
@@ -121,684 +112,250 @@ describe('validateOnlyCategories', () => {
 });
 
 describe('filterAuditsAndGroupsByOnlyOptions to be used in plugin config', () => {
-  it('should return given audits and groups if no only filter is set', () => {
-    const audits: Audit[] = [{ slug: 'speed-index', title: 'Speed Index' }];
-    const groups: Group[] = [
+  type PartialGroup = Partial<
+    Omit<Group, 'refs'> & { refs: Partial<Group['refs'][number]>[] }
+  >;
+  const basePluginConfig = (
```

Comment:
🧪 Maintainability: Is it possible to create tests with minimum audits or groups to avoid this need of abstraction and additional logic? What if each test only has 0-1 audit and 0-1 group?

Reply:
The schema parsing was involved. so I had to pass valid shapes.